Blockzilla/Modules/WebView/WebViewController.swift

```diff
@@ -139,18 +133,18 @@ class WebViewController: UIViewController, WebController {
     func stop() { browserView.stopLoading() }
 
     private func setupWebview() {
-        let wvConfig = WKWebViewConfiguration()
-        wvConfig.websiteDataStore = WKWebsiteDataStore.nonPersistent()
-        wvConfig.allowsInlineMediaPlayback = true
-        browserView = WKWebView(frame: .zero, configuration: wvConfig)
+        let configuration = WKWebViewConfiguration()
+        configuration.websiteDataStore = WKWebsiteDataStore.nonPersistent()
+        configuration.allowsInlineMediaPlayback = true
+        configuration.applicationNameForUserAgent = AppInfo.config.productName
```

Comment:
Did the apple documentation recommend this to be in the `Focus/38.0` style? If so then let's also add the version number.